util.py

```diff
@@ -335,16 +336,18 @@ def get_executable_path(
                 return path.args[0].join_right(path.args[1])
             else:
                 return tk.Path(path.get())
```

Comment:
in this line we are still calling `path.get()` so I am not sure if your intended changes can work this way.

Reply:
> But this exact case it the one thing that your PR is meant to fix.

ok, if we implement Alberts solution with ValueError, then I think all is solved.